lib/engine/game/g_18_ms/game.rb

```diff
@@ -175,14 +175,6 @@ class Game < Game::Base
         HEXES_FOR_GRAY_TILE = %w[C9 E11].freeze
         COMPANY_1_AND_2 = %w[AGS BS].freeze
 
```

Comment:
why did this change

Reply:
I removed line 178-184 as I noticed that these were not used anywhere. 
I can remove that change from the commit if you wish. Should I do that?

Reply:
@tobymao I have now removed the changed 18MS file - I can do that in a separate commit if I think it is good. Or are there some tool one can use to discover unused code?

Reply:
yo udidn't need to remove it, i was just asking